src/components/atoms/Cart.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+	<img v-bind:alt="cart.alt" v-bind:src="require(`../../assets/icons/${cart.src}`)" class="icons__cart">
```

Comment:
ditch the `v-bind` as Arturs mentioned. I think the shorthand version should work, too. E.g., `:alt="blablabla"` instead of `v-bind:alt="blablabla"`